### PR TITLE
Update UG to include monthly fee command

### PR DESCRIPTION
**Pull request by enhao25**
Only allow year between 1970 to 2038 to ensure that 2038 error doesn't happen (https://en.wikipedia.org/wiki/Year_2038_problem).

Ensures that people don't anyhow try to break the program with year 99999 or something.

### CODE REVIEW

**Review by JonahhGohh — APPROVED** [review]
LGTM!
So, this will only be an issue if the user is using a 32-bit system?

**Comment by enhao25** [comment]
> LGTM!
> So, this will only be an issue if the user is using a 32-bit system?

Yes I did a rough search and it seems that it is possible that 32-bit system running java might have the 2038 error. Hence I think setting an upper limit to <=2037 would be a good compromise.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -225,3 +240,9 @@ Action | Format, Examples
 **Find** | `find_session KEYWORD`<br><br>e.g. `find_session John`
 **Add** | `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`<br><br> e.g. `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 **Delete** | `delete_session n/STUDENT_NAME i/SESSION_INDEX`<br><br>e.g. `delete_session n/John Lee i/1`
+
+**Others**
```
Comment (samleewy) [inline docs/UserGuide.md]: I think the header can be "Fees"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -225,3 +240,9 @@ Action | Format, Examples
...
+
+Action | Format, Examples
+--------|------------------
+**fee** | `fee n/STUDENT_NAME m/MONTH y/YEAR`<br><br>e.g. `fee n/John Lee m/1 y/2021`
```
Comment (samleewy) [inline docs/UserGuide.md]: Then under action "Student fee by month", what do you think?

**Review by samleewy — COMMENTED** [review]
Some comments!

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Haha yeah I tot of before, but then I think at most there will only be 2 commands, one to get monthly fee for a particular month, one to get past 3 months fee. So not sure if it should be a section by itself. Although I think doesn't hurt to change ba. I will make the changes first then.

File: docs/UserGuide.md (hunk 2 above)
Comment (enhao25) [reply docs/UserGuide.md]: Actually not sure if its ambiguous, I think if we really want to be specific "Student fee for a particular month and year" would be more accurate. Will update to this version first.

**Review by samleewy — APPROVED** [review]
Approved with #91 for further discussion in future when there are possibly more fees command! 🎉

**Pull request merged**